cymetric/convenient_interface.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+
+from pyne import nucname
+import pandas as pd
+import numpy as np
+import cymetric as cym
+import warnings
+
+def merge_n_drop(pdf, base_col, add_pdf, add_col):
+    """
+    Merge some additionnal columns fram an additionnal Pandas Data Frame
+    onother one and then remove the second base column (keeping SimID
+    information).
+    Parameters
+    ----------
+    pdf : Pandas Data Frame
+    base_col : list of the base columns names
+    add_pdf : Pandas Data Frame to add in the pdf one
+    add_col : columns tobe added
+    """
+    pdf = pd.merge(add_pdf[add_col], pdf, on=base_col)
+    pdf.drop(base_col[1], 1)
+    return pdf
+
+def get_reduced_pdf(pdf, rdc_list):
+    """
+    Filter the pdf Pandas Data Frame according to the rdc_list (list of item
+    in the corresponding columns).
+    Parameters
+    ----------
+    pdf : Pandas Data Frame
+    rdc_list : list of pair of string and string list.
+    """
+    for rdc in rdc_list:
+        if len(rdc[1]) != 0:
+            pdf = pdf[pdf[rdc[0]].isin(rdc[1])]
+        else:
+            wng_msg = "Empty list provided for " + rdc[0] + " key."
+            warnings.warn(wng_msg, UserWarning)
+    return pdf
+
+def get_transaction_pdf(db, send_list=[], rec_list=[]):
+    """
+    Filter the Transaction Data Frame on specific sending facility and
+    receving facility.
+
+    Parameters
+    ----------
+    db : database
```

Comment:
Why not have people pass in the evaluator, rather than the db? There is not need to create a new evaluator each time this is called